editor/src/clj/editor/texture_set.clj

```diff
@@ -67,28 +73,34 @@
 
 (defn- flat-array->2d-points [flat-array]
   (into []
-        (partition-all 2)
+        (coll/partition-all-primitives :double 2)
         flat-array))
 
 (defn- ->anim-frame-from-geometry
-  [page-index frame-geometry]
+  [frame-index page-index tex-coords tex-coord-order frame-geometry]
   {:page-index page-index
-   :tex-coords (flat-array->2d-points (:uvs frame-geometry))
+   :tex-coords (->uv-quad frame-index tex-coords tex-coord-order)
    :vertex-coords (flat-array->2d-points (:vertices frame-geometry))
+   :vertex-tex-coords (flat-array->2d-points (:uvs frame-geometry))
```

Comment:
We need both the `:tex-coords` for the entire quad and the `:vertex-tex-coords` for the trimmed geometry. We need the untrimmed quad if an image is rendered in slice-9 mode.